src/tokens-unwrapped/tokens-unwrapped.entity.ts

```diff
@@ -25,6 +25,9 @@ export class TokensUnwrappedEntity {
   @Column({ type: 'numeric', precision: 38, scale: 0 })
   amount: string;
 
+  @Column({ type: 'numeric', precision: 38, scale: 0 })
```

Comment:
```suggestion
  @Column({ unique: 'true' })
```

We need to add a  unique constrain to avoid any potential duplicates. Also since this is a nonce we can use a number here. Typically we use numeric type for very large numbers.